src/triton_object_recognition/config/yolov3.yaml

```diff
@@ -0,0 +1,98 @@
+#There is an issue with ComposableNode where it doesn't understand the typical nested YAML structure
```

Comment:
Maybe name this something more descriptive of the fact that it loads the base yolo model, like 'base_yolov3.yaml'